### ISSUE TITLE
Sudden displaying error...

### ISSUE THREAD

**Issue by dpan7** [vasturiano/three-forcegraph#9]
All of the sudden, I am facing an error with imports

`import ThreeForceGraph from 'three-forcegraph';`

**Could not find a declaration file for module 'three-forcegraph'. /node_modules/three-forcegraph/dist/three-forcegraph.common.js' implicitly has an 'any' type.
  Try `npm install @types/three-forcegraph` if it exists or add a new declaration (.d.ts) file containing `declare module 'three-forcegraph';`**

All my nodes are displayed in the center stacked on one after another (I have visual representation but not the correct one)

Has something changed with the library? Also when I used to click on **ThreeForceGraph** it redirected to the component. No, it does nothing.

I much appreciate your help.

**Comment by vasturiano** [comment]
Hi @dpan7, hmmm it seems you are attempting to import the library in a TypeScript context, and therefore you're getting the error that the `.d.ts` declaration file does not exist. That's kind of expected. What's odd is that this file never existed in this library, so I don't see how it can be a new error. 
Have you changed something in your build system recently that may be contributing to this behavior changes? Different devDependencies versions perhaps?

**Comment by dpan7** [comment]
Hi there, I found the solution. I snippet of code was missing, that was the reason why the graph did not work.

Still I am getting this warning, but the graph works.. I assumed that it was the project's package version since I moved from 1.9 to 1.11. 

However, the solution was due to VScode version. VScode recently had a major update (1.22), so I tested in a environment prior to that version and no warnings displayed.

So for now, I will ignore the warnings :-) ...

Again thank you for help.
